CRITICAL - 2020-12-22 00:38:22 --> Cannot add or update a child row: a foreign key constraint fails (`ventas_database`.`cajas`, CONSTRAINT `cajas_ibfk_1` FOREIGN KEY (`id_usuario`) REFERENCES `usuario` (`Id`))
#0 D:\xampp\htdocs\SistemaVentas\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `cajas` ...')
#1 D:\xampp\htdocs\SistemaVentas\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `cajas` ...')
#2 D:\xampp\htdocs\SistemaVentas\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `cajas` ...')
#3 D:\xampp\htdocs\SistemaVentas\system\Database\BaseBuilder.php(2456): CodeIgniter\Database\BaseConnection->query('UPDATE `cajas` ...', Array, false)
#4 D:\xampp\htdocs\SistemaVentas\system\Model.php(889): CodeIgniter\Database\BaseBuilder->update()
#5 D:\xampp\htdocs\SistemaVentas\app\Controllers\Caja.php(36): CodeIgniter\Model->update(Array, Array)
#6 D:\xampp\htdocs\SistemaVentas\system\CodeIgniter.php(918): App\Controllers\Caja->cerrar()
#7 D:\xampp\htdocs\SistemaVentas\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Caja))
#8 D:\xampp\htdocs\SistemaVentas\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\SistemaVentas\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-12-22 00:38:22 --> Uncaught ErrorException: Undefined variable: viewFile in D:\xampp\htdocs\SistemaVentas\system\Debug\Exceptions.php:308
Stack trace:
#0 D:\xampp\htdocs\SistemaVentas\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'D:\\xampp\\htdocs...', 308, Array)
#1 D:\xampp\htdocs\SistemaVentas\system\Debug\Exceptions.php(308): include()
#2 D:\xampp\htdocs\SistemaVentas\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(mysqli_sql_exception), 500)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(mysqli_sql_exception))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-12-22 00:39:06 --> Cannot add or update a child row: a foreign key constraint fails (`ventas_database`.`cajas`, CONSTRAINT `cajas_ibfk_1` FOREIGN KEY (`id_usuario`) REFERENCES `usuario` (`Id`))
#0 D:\xampp\htdocs\SistemaVentas\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `cajas` ...')
#1 D:\xampp\htdocs\SistemaVentas\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `cajas` ...')
#2 D:\xampp\htdocs\SistemaVentas\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `cajas` ...')
#3 D:\xampp\htdocs\SistemaVentas\system\Database\BaseBuilder.php(2456): CodeIgniter\Database\BaseConnection->query('UPDATE `cajas` ...', Array, false)
#4 D:\xampp\htdocs\SistemaVentas\system\Model.php(889): CodeIgniter\Database\BaseBuilder->update()
#5 D:\xampp\htdocs\SistemaVentas\app\Controllers\Caja.php(36): CodeIgniter\Model->update(Array, Array)
#6 D:\xampp\htdocs\SistemaVentas\system\CodeIgniter.php(918): App\Controllers\Caja->cerrar()
#7 D:\xampp\htdocs\SistemaVentas\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Caja))
#8 D:\xampp\htdocs\SistemaVentas\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\SistemaVentas\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-12-22 00:39:07 --> Uncaught ErrorException: Undefined variable: viewFile in D:\xampp\htdocs\SistemaVentas\system\Debug\Exceptions.php:308
Stack trace:
#0 D:\xampp\htdocs\SistemaVentas\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'D:\\xampp\\htdocs...', 308, Array)
#1 D:\xampp\htdocs\SistemaVentas\system\Debug\Exceptions.php(308): include()
#2 D:\xampp\htdocs\SistemaVentas\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(mysqli_sql_exception), 500)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(mysqli_sql_exception))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-12-22 00:41:19 --> Cannot add or update a child row: a foreign key constraint fails (`ventas_database`.`cajas`, CONSTRAINT `cajas_ibfk_1` FOREIGN KEY (`id_usuario`) REFERENCES `usuario` (`Id`))
#0 D:\xampp\htdocs\SistemaVentas\system\Database\MySQLi\Connection.php(331): mysqli->query('UPDATE `cajas` ...')
#1 D:\xampp\htdocs\SistemaVentas\system\Database\BaseConnection.php(741): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `cajas` ...')
#2 D:\xampp\htdocs\SistemaVentas\system\Database\BaseConnection.php(669): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `cajas` ...')
#3 D:\xampp\htdocs\SistemaVentas\system\Database\BaseBuilder.php(2456): CodeIgniter\Database\BaseConnection->query('UPDATE `cajas` ...', Array, false)
#4 D:\xampp\htdocs\SistemaVentas\system\Model.php(889): CodeIgniter\Database\BaseBuilder->update()
#5 D:\xampp\htdocs\SistemaVentas\app\Controllers\Caja.php(36): CodeIgniter\Model->update(Array, Array)
#6 D:\xampp\htdocs\SistemaVentas\system\CodeIgniter.php(918): App\Controllers\Caja->cerrar()
#7 D:\xampp\htdocs\SistemaVentas\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Caja))
#8 D:\xampp\htdocs\SistemaVentas\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\SistemaVentas\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2020-12-22 00:41:19 --> Uncaught ErrorException: Undefined variable: viewFile in D:\xampp\htdocs\SistemaVentas\system\Debug\Exceptions.php:308
Stack trace:
#0 D:\xampp\htdocs\SistemaVentas\system\Debug\Exceptions.php(308): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'D:\\xampp\\htdocs...', 308, Array)
#1 D:\xampp\htdocs\SistemaVentas\system\Debug\Exceptions.php(308): include()
#2 D:\xampp\htdocs\SistemaVentas\system\Debug\Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(mysqli_sql_exception), 500)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(mysqli_sql_exception))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
